### ISSUE TITLE
Clarification on "validation set of ILSVRC 2012 - 2016 classiifcation task"

### ISSUE THREAD

**Issue by vnalluri** [cvjena/cnn-models#8]
Hello
Very nice work. I am trying to reproduce the error rates for Alexnet. 
Before I get started, I thought I will get a clarification on the training and validation data set you have used. There was a mention of ILSVRC 2012-2016 classification task in the "Accuracy on ImageNet" section, does that mean
1. The accuracy mentioned is cummulative accuracy across the validation data-sets from 2012 to 2016, 4 different validation sets? If yes, then was the training also performed on 4 different data-sets?

2. The train_log seems to be generated accross 5 days, which I think is the training time for Alexnet on the 1.2million dataset. So was the training performed on 2012 training data-set and accuracy measured across 2012-2016 validation datasets?

Thanks
Vamsi

**Comment by MarcelSimon** [comment]
That classification task was unchanged in these years. Hence by ILSVRC 2012-2016 we referred to the ILSVRC2012 classification task.
